cmd/argocd/commands/cluster.go

```diff
@@ -44,8 +44,11 @@ func NewClusterCommand(clientOpts *argocdclient.ClientOptions, pathOpts *clientc
 // NewClusterAddCommand returns a new instance of an `argocd cluster add` command
 func NewClusterAddCommand(clientOpts *argocdclient.ClientOptions, pathOpts *clientcmd.PathOptions) *cobra.Command {
 	var (
-		inCluster bool
-		upsert    bool
+		inCluster      bool
+		upsert         bool
+		awsAuth        bool
```

Comment:
I think awsAuth bool is redundant. We can infer AWS auth by the presence of awsClusterName.